cmake/ctypesgen.cmake

```diff
@@ -0,0 +1,63 @@
+#[[
+AUTHOR(S):  Rashad Kanavath <rashad km gmail>
+PURPOSE:    Cmake building of lib/python/ctypes (TODO)
+COPYRIGHT:  (C) 2020 by the GRASS Development Team
+            This program is free software under the GPL (>=v2)
+            Read the file COPYING that comes with GRASS for details.
```

Comment:
Update copyright etc.

Reply:
There are many `CMakeLists.txt` and `*.cmake` files. Do we want this block in every file?

Reply:
I think one license file per project should be enough, but the best practice is different. We include documentation and selected authors, so that's sometimes useful. (I guess one way how to think about the authors is that these are the people to contact about the source code.)

Reply:
(But we don't need the text in external files and I think we don't keep this for each file actually, just the main file for each directory if there is such file like for tools.)

Reply:
We may include a SPDX-License-Identifier line as supposed below.

Reply:
> We may include a SPDX-License-Identifier line as supposed below.

The rest is optional I think, not _really_ necessary.